tests/cli/test_datasource_athena.py

```diff
@@ -104,7 +105,6 @@ def test_cli_athena_datasource_new_connection_string(
     region_test = "us-east-1"
     schema_test = "test_data"
     table_name_test = "table_test"
-    connection_string = f"awsathena+rest://@athena.{region_test}.amazonaws.com/{schema_test}?s3_staging_dir=s3://YOUR_S3_BUCKET/path/to/"
```

Comment:
Was this deleted on purpose?

Reply:
ruff removes variables if they aren't referenced.
To prevent this behavior I'll need to either ignore the warning or save the variable with a `_`.
Example:  `_ = "foobar"`.

I'm going to update this one to ignore the warning.

Reply:
Added back with `_`.
If the original intention was to use the `connection_string` for something that can be added later.